mobile_files/package.json.orig

```diff
@@ -54,7 +54,7 @@
     "react-native-safe-area-view": "0.9.0",
     "react-native-securerandom": "git+https://github.com/status-im/react-native-securerandom.git#0.1.1-2",
     "react-native-splash-screen": "3.1.1",
-    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#v2.3.1",
+    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#v2.3.2",
```

Comment:
Did you commit this file by accident?

Reply:
Oh no, just double checked as the "orig" suffix kind of looked like a backup file of text editor. Never mind that. Thanks

Reply:
@goranjovic it confused me too, but afaik we now use this file since migration to `yarn`